src/components/MainDisplay.js

```diff
@@ -74,6 +79,11 @@ class MainDisplay extends React.Component {
       showIndividual={this.displayIndividual}/>)
     return (
       <div>
+        {this.state.errorMessage.length &&
+          <div>
+        <h1 className="error">{this.state.errorMessage}</h1>
+        <button onClick={this.componentDidMount}>BACK</button>
+          </div>}
```

Comment:
Great idea to add a button so they can navigate back to the main page and try a different movie.